index.html

```diff
@@ -15,31 +15,31 @@
     <header>
       <section class="top-nav-bar">
         <ul class="upper-nav">
-          <a href=""><i class="fa fa-facebook" aria-hidden="true"></i></a>
-        </li>
-        <li>
-          <a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a>
-        </li>
-        <li><a href="">Logout</a></li>
-        <li><a href="">My Page</a></li>
-      </ul>
+          <li>
+            <a href=""><i class="fa fa-facebook" aria-hidden="true"></i></a>
```

Comment:
Kindly, I suggest you put "#" in the HREF attributes, this will make your code look more professional.